docs/migration/v11.x.md

```diff
@@ -40,68 +41,56 @@ publishing.
 During each release stage, we're looking to get feedback from teams in our
 [Release Partner Program](https://github.com/carbon-design-system/carbon/wiki/Release-Partner-Program).
 
-## Previewing the release
-
-There are two ways to try out the v11 release of Carbon:
-
-- Feature flags along with the `carbon-components` and `carbon-components-react`
-  packages
-- Using the new packages directly, like `@carbon/react` or `@carbon/styles`
-
-As we near the v11 release, the preference will be to use the `@carbon/react`
-package. However, for the alpha and beta release, full compatibility will not be
-available, so we recommend trying out new features using feature flags.
-
-### Feature flags
-
-Our team makes use of
-[feature flags](https://martinfowler.com/articles/feature-toggles.html) to ship
-new code in our v10 release without breaking existing usage.
+## Updates
 
-To try out features related to the v11 release, you will use the
-`enable-v11-release` flag in both Sass and JavaScript.
+### Single-entrypoint packages
 
-To enable this flag in `carbon-components`, you will need to set
-`$feature-flags` before importing `carbon-components`. For example:
+Starting in v11, we will prefix many of the packages we ship with the `@carbon`
+scope. We will also try and consolidate packages where possible to get as close
+to a one-package, one-install experience.
 
-```scss
-$feature-flags: (
-  enable-v11-release: true,
-);
+If you're using React today, you need to install several packages:
```

Comment:
```suggestion

```

Maybe removing this sentence and 3 packages

I felt like this little blurb was a bit confusing as a reader, and instead going straight line 58. `In v11....etc.`